packages/inter-protocol/src/vpool-xyk-amm/pool.js

```diff
@@ -92,7 +90,14 @@ export const definePoolKind = (baggage, ammPowers, storageNode, marshaller) => {
    * @returns {ImmutableState & MutableState}
    */
   const poolInit = (liquidityZcfMint, poolSeat, secondaryBrand) => {
-    /** @type {PublishKit<{secondary: Amount, central: Amount}>} */
+    /** @type {StoredPublishKit<InitPublication>} */
+    const { publisher: initPublisher } = makeStoredPublishKit(
+      E(storageNode).getChildNode('init'),
```

Comment:
Can we mock a storage node and/or makeStoredPublishKit for a test?

Reply:
https://github.com/Agoric/agoric-sdk/pull/5948